frontend/www/js/omegaup/components/problem/creator/Header.vue

```diff
@@ -21,7 +21,7 @@
           {{ T.problemCreatorGenerateProblem }}</span
         >
       </b-button>
-      <b-button variant="warning" size="sm">
+      <b-button variant="warning" size="sm" @click="createNewProblem()">
```

Comment:
You need to pass a function reference, not a function call as the callback for the click event.
```suggestion
      <b-button variant="warning" size="sm" @click="createNewProblem">
```

Reply:
Hi @carlosabcs, may you look into this SO [answer](https://stackoverflow.com/questions/62828904/vue-testing-jest-button-triggerclick-not-working#:~:text=i%20have%20been%20struggling%20with%20this%20as%20well%2C%20it%20seems%20sometimes%20the%20test%20component%20has%20a%20hard%20time%20finding%20the%20emitting%2Fcalls%20to%20functions%20that%20do%20not%20have%20parentheses.)? For testing purposes, I added the parenthesis.

Reply:
Before accepting this, and because it looks weird. Could you try using `mount` instead of `shallowMount`? See reference [here](https://stackoverflow.com/questions/51536537/running-jest-with-bootstrap-vue)

Reply:
However, maybe you need to add:
`localVue.use(BootstrapVue)`.

Or did you try this: https://stackoverflow.com/a/53598621? Please try other options and if they don't finally work. Then let's use what you did because passing a function call seems uncommon.

Reply:
The problem with `shallowMount` is that it only renders the current component, the nested components are not "compiled". That's why BootstrapVue component's are not being correctly rendered.